experimenter/experimenter/features/manifests/nimbus_fml_loader.py

```diff
@@ -1,37 +1,60 @@
-import logging
 import os
 
 import yaml
+from packaging import version as Version
 from rust_fml import FmlClient
 
 from experimenter.settings import BASE_DIR
 
-logger = logging.getLogger()
-
 
 class NimbusFmlLoader:
-    def __init__(self, application: str, channel: str):
+    __base_path = os.path.join(BASE_DIR, "features", "manifests", "apps.yaml")
```

Comment:
Does this need to use a double underscore? `__symbol` is only necessary to mangle it for subclass inheritance (e.g., this symbol will get mangled to `_NimbusFmlLoader__base_path`).

Reply:
ahh that makes sense. Uppercase it is! 🙌